Sample-01/src/config.js

```diff
@@ -0,0 +1,18 @@
+import configJson from "./auth_config.json";
+
+// Configure the audience here. By default, it will take whatever is in the config
+// (specified by the `audience` key) unless it's the default value of "YOUR_API_IDENTIFIER" (which
+// is what you get sometimes by using the Auth0 sample download tool from the quickstart page, if you
+// don't have an API).
+// If this resolves to `null`, the API page changes to show some helpful info about what to do
+// with the audience.
+const audience =
+  configJson.audience && configJson.audience !== "YOUR_API_IDENTIFIER"
+    ? configJson.audience
+    : null;
+
+export const config = {
+  domain: configJson.domain,
+  clientId: configJson.clientId,
+  ...audience,
```

Comment:
Why is audience being spread here? Isn't it just a string (or null)?

Reply:
`audience` might be `null`; if it is, spreading it will omit it from the object rather than inserting `audience: null` into the object.

Reply:
https://stackoverflow.com/a/47296650

Reply:
I could include a reference link in the source code, as I agree it's a bit of a niche syntax in this scenario.

Reply:
But audience can also be a string ... spreading a string isn't what we want here, I think...  https://jsfiddle.net/3nrvhaxm/

Reply:
I think you want ...(audience ? { audience } : null),

Reply:
🤦 I had forgotten about the case where you _do_ have an audience - will fix!